packages/atomic/cypress/integration/facets/facet-common-actions.ts

```diff
@@ -11,31 +11,103 @@ export function selectIdleCheckboxValueAt(
   FacetWithCheckboxSelector: FacetWithCheckboxSelector,
   index: number
 ) {
-  FacetWithCheckboxSelector.idleCheckboxValue().eq(index).click();
+  FacetWithCheckboxSelector.idleCheckboxValueLabel()
+    .eq(index)
+    .then((idleCheckboxValueLabel) => {
+      const hasAriaLabel = !!idleCheckboxValueLabel.attr('aria-label');
+      const text = hasAriaLabel
+        ? idleCheckboxValueLabel.attr('aria-label')!
+        : idleCheckboxValueLabel.text();
+      FacetWithCheckboxSelector.idleCheckboxValueLabel()
+        .filter(
+          hasAriaLabel ? `[aria-label="${text}"]` : `:contains("${text}")`
+        )
+        .its('length')
+        .should(
+          'eq',
+          1,
+          'There should not be any other value similar to this one.'
+        );
+      cy.wrap(idleCheckboxValueLabel).click();
```

Comment:
Like with the actions I introduced last time for category facets, we need to call cypress's `click` instead of the native one. Otherwise, focus is lost.